doc/configuration.md

```diff
@@ -306,7 +305,8 @@ jack:///?name=Channel4&sampleformat=48000:16:1&autoconnect=system:playback_&auto
 
 - The `chunk_ms` parameter is ignored for jack streams. Instead, the Jack
   buffer size (Frames/Period) is used. So if you are encoding your streams with
-  Ogg, ensure that the Jack server Frames/Period size is set appropriately.
+  anything other than "pcm", ensure that the Jack server Frames/Period size is
```

Comment:
I don't understand this sentence. Should the user configure the frame size to the codec delay/buffer? I think this buffer is sometimes even dynamic, if you want to keep the overall latency low, you should feed small chunks into the codec (e.g. 10ms frames). This would also be valid for resampling, soxr introduces also a delay. 
I would just remove this sentence to avoid confusion.

Reply:
Done. I think I got confused by the following sentence in the default `snapserver.conf`: "Some codecs have a higher latency and will need more data, e.g. Flac will need ~26ms chunks"